### PR TITLE
Applying concepts in a Newsweek clone page

### PR DESCRIPTION
**Pull request by kenderb**
The page contains the next mains tags:

- A header tag that contains the navbar with some buttons.
- Combination of the Grid system and bootstrap classes.
- Contain media Query for 2 breakpoints.
- A footer that contains some lists and images.

### CODE REVIEW

File: styles/main.css
Comment (javitocor) [inline styles/main.css:339]: Apply bootstrap classes wherever you can, please add bootstrap classes and reduce your CSS file.

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team!
Very good job in this project! 👍 
Linters are green 🟢 
You set up bootstrap correctly 👏 
Professional readme and descriptive commit messages 💯 
I am impressed! 🥇 

However, we still need to implement some changes to get this project approved 💪 
Kindly check my comments below 👇  apply the changes and resubmit your project for a new code review
Good luck 😺 

## Mandatory Changes
- Kindly, modify font size and spacing in all the project, make it equal to the original
- In the editors pick section the topic is missing, kindly add it
Capture1
Capture
- Screens below 990px:
   -  kindly fix the size of the columns in the US, business, sports... sections, it is not corresponding with the original
Capture3
Capture2

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__This is an excellent piece of work. It is so obvious that a lot of efforts have been put into the development of this project.  I can not approve it yet because there are some changes that are required, but overall, your project is looking good. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Ensure that the height of the `SUBSCRIBE` section is the same as that of the `NEWSLETTER` section. This is in line with what we have on the original page.
- You should also adjust the width of the `SUBSCRIBE` section so that it's reduced to look like what we have on the original page.
Original page:


Your Clone page:


- [x] The `US, WORLD, BUSINESS, TECH & SCIENCE, CULTURE, AND SPORTS` section are not displayed in small screen sizes (<768px) as seen in the original. Kindly hide them for this specific screen size.

**[ OPTIONAL CHANGES ]**
- As the main point of this project is to use Bootstrap, CSS rules like padding, margin, width, etc all have dedicated class names in bootstrap. You are advised to use Bootstrap classes for these features.


**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:13]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:23]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:43]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:51]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:59]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:97]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:102]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:106]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

File: styles/main.css
Comment (arslanbisharat) [inline styles/main.css:110]: Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [x]  Please add these folders into a single folder. This will help you to present your project professionally. 
chrome-capture (68)

- [x] Please add image into its respective folder
chrome-capture (69)

- [ ]  Bootstrap projects make sure to less use the styles.css file for styles that could be done using Bootstrap classes since that's the main goal of the exercise. Things like font-weight, flex justifications, display, etc could be implemented completely with Bootstrap classes. [Please refer to the main page of bootstrap to learn more about bootstrap. ](https://getbootstrap.com/docs/4.5/getting-started/introduction/)
I have given a few suggestions below. Please have a look at the entire CSS file and change all the extra styling to bootstrap. In case you cannot implement a change, please leave that in the CSS file.  

- [ ] **This is incredible that you have a sticky navbar in your project which is not an easy task to do and most students could not achieve it. But on the original page, on scrolling, the navbar decreases its width, the date, location, and temperate disappears and the Newsweek logo takes their place.**   

**Clone**

chrome-capture (57)

**Original**

chrome-capture (58)

- [x]  **`FEATURED STORIES, TOP STORY, THE DEBATE` should be aligned in the same row(In case of Tablet Mode only . Please make this correct.** 

**Clone** 

chrome-capture (60)

**Original**

chrome-capture (61)

- [x] **There is no white space on the original page before the ` IN THE MAGAZINE` section. Please have a look at the original page and work on it.** 

**Clone**

chrome-capture (63)

**Original**

chrome-capture (62)

- [x] In the footer please make your logo a link. 

- [x] On the medium and small screen in the Newsletter section Free Signup button is overlapping the text. Please resolve this.

**Clone**

chrome-capture (64)

**Original**

chrome-capture (65)

- [ ] On the small and medium screen original page in the **Subscribe** section, the text has more height than the clone. Please have a look below.

**Original** 

chrome-capture (66)

**Clone**

chrome-capture (67)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [ ] Bootstrap projects make sure to less use the styles.css file for styles that could be done using Bootstrap classes since that's the main goal of the exercise. Things like font-weight, flex justifications, display, etc could be implemented completely with Bootstrap classes. Please refer to the main page of bootstrap to learn more about bootstrap.
I have given a few suggestions below. Please have a look at the entire CSS file and change all the extra styling to bootstrap. In case you cannot implement a change, please leave that in the CSS file.

                                                        ### Large Screen

- [ ] Newsletter and subscribe section has not met the minimum requirement to approve this project. Please work on it. 

**Clone**

chrome-capture (70)

**Original**

chrome-capture (71)

- [x] Newsweek logo and social icons are not correctly aligned. And the space between the line and this section is way to more. 

**Clone**

chrome-capture (72)


**Original**

chrome-capture (73)

                       ### Medium Size

- [x] On tablet mode text is overflowing in some sections. 

**Clone**

chrome-capture (74)

**Original**

chrome-capture (75)

- [x] Position of the DISSENT section is not correct as present on the original page. 

**Clone**

chrome-capture (76)

**Original**

chrome-capture (77)


- [ ] In the subscribe section, the text and button are not present in their actual positions. I have given suggestions below. 

**Clone**

chrome-capture (78)

**Original**

chrome-capture (79)

- [ ] Subscribe section and newsletter section should have the same width. 

**Clone**

chrome-capture (80)

**Original**

chrome-capture (81)

                                      ### Small Screen

- [x] You have two breakpoints after opinion section. Please work on it as on the original page there is only one. 

**Clone**

chrome-capture (84)

**Original**

chrome-capture (85)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾 (⚠)

### I really like your project! Congrats team, you guys did an excellent job, you have my approval to go ahead and merge this pull request!

_**Clean and modular code, following the best practices, and the design looks great, keep it up!**_ 🥇

[OPTIONAL CHANGES]

**⚠These are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:**
- [ ] Your navbar should be fixed in large screens;


## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**
